utils/scripts/install-centos8.sh

```diff
@@ -68,4 +69,12 @@ dnf install \
     sg3_utils \
     valgrind-devel \
     yasm
+
+if [ "$fedora_java" = "yes" ]; then
+	dnf install java-1.8.0-openjdk-devel
+	curl --output ./apache-maven-3.6.3-bin.tar.gz https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz && tar -xf apache-maven-3.6.3-bin.tar.gz && find ./ -name "mvn" | { read d; f=$(dirname $d); cp -r $(readlink -f $f)/../* /usr/local; }
```

Comment:
(style)  line over 100 characters
(lint) read without -r will mangle backslashes. [SC2162]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Double quote to prevent globbing and word splitting. [SC2086]